### PR TITLE
Soft crit

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: 
balance: You no longer take damage/are as heavily blinded in crit while above -30HP
balance: Whispering in crit above -30HP will not cause you to succumb
balance: You can now hear in crit above -30HP
/:cl:

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -37,36 +37,36 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
...
-	"�" = "admin",
-	"�" = "deadmin",
+	"�" = "admin",
+	"�" = "deadmin",
 
 	// Misc
-	"�" = "AI Private",
-	"�" = "cords"
+	"�" = "AI Private",
+	"�" = "cords"
```
Comment (ChangelingRain) [inline code/modules/mob/living/say.dm]: uh oh

**Comment by KorPhaeron** [comment]
Nice thanks

File: code/modules/mob/living/say.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/mob/living/say.dm]: Fuck

**Comment by nicbn** [comment]
30 HP from death or from crit? If from death, use -70 HP instead, as you die when you reach -100 HP, and not 0 HP.

**Comment by nicbn** [comment]
It's not 30 HP, it's -30 HP. Writing 30 HP is really confusing, as 30 HP from death means -70 HP.
Write instead -30 HP or 70 HP from death.

**Comment by Cyberboss** [comment]
My bad, fixed cl

File: code/modules/mob/living/say.dm (hunk 1 above)
Comment (nicbn) [inline code/modules/mob/living/say.dm]: > This file uses cp1251 as encoding.

**Comment by Xhuis** [comment]
Does this also force you to crawl and be unable to use items like Goon has (or used to, last I checked?) Either way, I'm curious how this'll play out.

**Comment by Cyberboss** [comment]
You're still effectively UNCONSCIOUS as far as game code is concerned, so you can't do anything

**Comment by optimumtact** [comment]
didn't goof do this?

**Comment by Xhuis** [comment]
Evidently, if he did, he was too busy _goofing off_

**Comment by Chronitonity** [comment]
iunno if hippie still has the thing where until you were at -50~ you could crawl and whisper, actually added a lot seeing people try and crawl a way from a traitor

**Comment by KorPhaeron** [comment]
Could try crawling that doesnt work if you are being pulled

**Comment by DoctoreMad** [comment]
Tip of the Round: Stunning and cuffing someone leaves you fragile to comebacks. If you have to arrest someone has a security officer, put them in soft crit and drag them around!

**Comment by KorPhaeron** [comment]
Dragging people while they are bleeding is still dangerous

**Comment by DoctoreMad** [comment]
If they die due to indirect bleeding, its not your fault anyways!

**Comment by KorPhaeron** [comment]
@Cyberboss Could you please add very slow movement in soft crit (that won't work when you are being pulled by someone else) or do you want me to do it in a followup PR

Could probably use some examine text that lets you know a carbon is in crit as well

**Comment by Steelpoint** [comment]
Its a nice step but this is really still a hard crit than a soft crit.

Soft crit is more when you are still able to move and talk, but typically have restricted vision, slurred speech (or unable to use radio) and move very slowly while being unable to use weapons.

**Comment by XDTM** [comment]
Far softer than being 100% powerless.

**Comment by Fox-McCloud** [comment]
Yeah, going to second what Steelpoint said. There really isn't much difference here.

If Goof would revive Goon crit and worked with a few others for weapons damage adjustments, that would be a vastly superior system.

The test pilot of the system didn't work because weapon damages weren't adjusted AND there was an active bug with it.

**Comment by Bawhoppen** [comment]
no goonmed didn't work because goof did it

**Comment by Steelpoint** [comment]
I think Gooncrit just lets you move around but you can't interact with the environment, also you move at a crawling speed. 

Can't recall, its been a while.

**Comment by Fox-McCloud** [comment]
>I think Gooncrit just lets you move around but you can't interact with the environment, also you move at a crawling speed.

Gooncrit is a complex system whereby you die due to weighted brain damage and O2 loss, with shock, cardiac arrest and cardiac failure all factoring in and making the whole thing more complicated; it's not complexity for complexity's sake though; it generates a lot of additional gameplay for medical and makes it feel like you have a "fighting chance" when you go into crit (even though you usually don't).

**Comment by Steelpoint** [comment]
Good to know Fox.

Crawling needs expansion to a extent, if you are stuck in a crawling state you are effectively dead since you can't pick up and use items.

Honestly I would pay money if someone ported goon's medical system over to a extent.

**Comment by Alek2ander** [comment]
It shouldn't be called _critical condition_ if it doesn't lead to death unless something is done to prevent it. As is, if you are in a rarely visited area, you can stay in softcrit indefinitely, stuck in an unplayable state, with succumbing (to what? you're not dying!) being the only way out.

**Comment by Cyberboss** [comment]
This will be back

**Comment by KorPhaeron** [comment]
@cyberboss can't we just merge this as is without crawling and do that later?

**Comment by nicbn** [comment]
Crawling is not that hard, is it?
I'm pretty sure we can port it from yog (if it stills have it).

**Comment by KorPhaeron** [comment]
Damn I was hoping I could just merge this

**Comment by KorPhaeron** [comment]
Wonder how many days before the revert PR

**Comment by GunHog** [comment]
Re-opening and fast merging a 2 month closed PR is kind of iffy practice...

**Comment by KorPhaeron** [comment]
Probably true

**Comment by PraiseRatvar** [comment]
Really true...

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -687,6 +687,9 @@
 			return
 		if(IsUnconscious() || IsSleeping() || getOxyLoss() > 50 || (status_flags & FAKEDEATH) || health <= HEALTH_THRESHOLD_CRIT)
 			if(stat == CONSCIOUS)
+				stat = SOFT_CRIT
+				update_canmove()
+			if(health <= HEALTH_THRESHOLD_FULLCRIT && stat == SOFT_CRIT)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/carbon.dm]: ha

**Pull request merged**